packages/dart_frog_test/example/routes/dice.dart

```diff
@@ -0,0 +1,18 @@
+import 'dart:io';
+import 'dart:math';
+
+import 'package:dart_frog/dart_frog.dart';
+
+Response onRequest(RequestContext context) {
+  return switch (context.request.method) {
+    HttpMethod.post => _onPost(context),
+    _ => Response(statusCode: HttpStatus.methodNotAllowed),
+  };
+}
+
+Response _onPost(RequestContext context) {
+  final rng = context.read<Random>();
```

Comment:
**Nit:** Rename to "random" over using abbreviated form for readability and comprehension (doesn't assume everyone knows that _rng = Random Number Generator_)
```suggestion
  final random = context.read<Random>();
```